src/main/java/mate/jdbc/service/impl/CarServiceImpl.java

```diff
@@ -0,0 +1,69 @@
+package mate.jdbc.service.impl;
+
+import java.util.List;
+import java.util.NoSuchElementException;
+import mate.jdbc.dao.CarDao;
+import mate.jdbc.lib.Inject;
+import mate.jdbc.lib.Service;
+import mate.jdbc.model.Car;
+import mate.jdbc.model.Driver;
+import mate.jdbc.service.CarService;
+
+@Service
+public class CarServiceImpl implements CarService {
+    @Inject
+    private CarDao carDao;
+
+    @Override
+    public Car create(Car car) {
+        return carDao.create(car);
+    }
+
+    @Override
+    public Car get(Long id) {
+        return carDao.get(id).orElseThrow(
+                () -> new NoSuchElementException("Don't found car with id: " + id)
+        );
+    }
+
+    @Override
+    public List<Car> getAll() {
+        return carDao.getAll();
+    }
+
+    @Override
+    public Car update(Car car) {
+        return carDao.update(car);
+    }
+
+    @Override
+    public boolean delete(Long id) {
+        return carDao.delete(id);
+    }
+
+    @Override
+    public void addDriverToCar(Driver driver, Car car) {
+        List<Driver> drivers = car.getDrivers();
+        drivers.add(driver);
+        car.setDrivers(drivers);
+        carDao.update(car);
+    }
+
+    @Override
+    public void removeDriverFromCar(Driver driver, Car car) {
+        List<Driver> drivers = car.getDrivers();
+        for (int i = 0; i < drivers.size(); i++) {
+            if (drivers.get(i).equals(driver)) {
+                drivers.remove(i);
+                break;
+            }
+        }
+        car.setDrivers(drivers);
+        carDao.update(car);
+    }

```

Comment:
It is better to do like:
```suggestion
    @Override
    public void removeDriverFromCar(Driver driver, Car car) {
        car.getDrivers().remove(driver);
        carDao.update(car);
    }
```